src/impl/sink_to_file.cpp

```diff
@@ -12,13 +12,61 @@
 #include <fmt/chrono.h>
 
 namespace soralog {
-  using namespace std::chrono_literals;
+
+  namespace {
+
+    using namespace std::chrono_literals;
+
+    constexpr std::string_view separator = "  ";
```

Comment:
Why is it two spaces, exactly?

Reply:
I mean, this seems oddly specific, maybe a comment here would be a good thing.

Reply:
Separator between logical parts of log record. Might be any substring or symbol: space, tab, etc. Couple of space is selected to differ of single space.